src/pages/cart-page/cart-control-product/cart-control-product.tsx

```diff
@@ -0,0 +1,80 @@
+import { IconButton, useMediaQuery } from "@mui/material";
+import { Dispatch, FC } from "react";
+import RemoveOutlinedIcon from "@mui/icons-material/RemoveOutlined";
+import AddOutlinedIcon from "@mui/icons-material/AddOutlined";
+import { CartProductModel } from "../../../api/api-cart/api-get-cart";
+import { useCartContext } from "../../../context/use-cart-context";
+import { ControlBox, CountBox, TypographyQuantity } from "../cart-page.styled";
+import { breakpointViewSize } from "../../../state/state";
+
+interface DrawerCartBoxItemControlInterface {
+  setCartItemQuantity: Dispatch<React.SetStateAction<number>>;
+  cartItemQuantity: number;
+  cartItem: CartProductModel;
+}
+
+const DrawerCartBoxItemControl: FC<DrawerCartBoxItemControlInterface> = ({
+  cartItemQuantity,
+  setCartItemQuantity,
+  cartItem,
+}) => {
+  const cartContext = useCartContext();
+
+  const countCartProduct = (action: string) => {
+    cartContext.setIsActive(!cartContext.isActive);
+    if (action != "addition") {
```

Comment:
Why != instead of !== ?

Reply:
Mozna uzyć obydwóch form. Druga forma będzie dogłebnie sprawdzała oprócz typu równiez zawartość. Wybrałem pierwszą formę poniewaz chciałem uniknąć błędów.